tests/components/controlBar-test.js

```diff
@@ -49,78 +47,39 @@ describe('ControlBar', function() {
     baseMockProps = {
       isLiveStream: false,
       controller: baseMockController,
-      skinConfig: JSON.parse(JSON.stringify(defaultSkinConfig))
+      skinConfig: defaultSkinConfig
```

Comment:
@KseniiaSokolovaEpam sorry, I'm just seeing this now. We can fix this later, but `Utils.clone` returns a shallow copy of the object, so if you modify any nested properties of the skin config they will be modified for all test cases. This means that a test can affect the outcome of the next one.

We should keep the `JSON.parse(JSON.stringify` line. It's inefficient, but for the purpose of unit tests that doesn't matter.